tests/FSharp.Test.Utilities/Compiler.fs

```diff
@@ -1156,30 +1158,48 @@ Actual:
             | Some p -> func p il
         | CompilationResult.Failure f -> failwith $"Result should be \"Success\" in order to get IL. Failure: {Environment.NewLine}{f}"
 
+    let withILContains expected result : CompilationResult =
+        match result with
+        | CompilationResult.Success s ->
+            match s.OutputPath with
+            | None -> failwith "Operation didn't produce any output!"
+            | Some p ->
+                match ILChecker.verifyILAndReturnActual [] p expected with
+                | true, _, _ -> result
+                | false, errorMsg, _actualIL -> CompilationResult.Failure( {s with Output = Some (ExecutionOutput { StdOut = errorMsg; ExitCode = 0; StdErr = "" })} )
+
+        | CompilationResult.Failure f -> failwith $"Result should be \"Success\" in order to get IL. Failure: {Environment.NewLine}{f}"
+
     let verifyIL = doILCheck ILChecker.checkIL
 
     let verifyILNotPresent = doILCheck ILChecker.checkILNotPresent
 
     let verifyILBinary (il: string list) (dll: string)= ILChecker.checkIL dll il
 
-    let private verifyFSILBaseline (baseline: Baseline) (result: CompilationOutput) : unit =
-        match result.OutputPath with
-        | None -> failwith "Operation didn't produce any output!"
-        | Some p ->
-            let expectedIL =
-                match baseline.ILBaseline.Content with
-                | Some b -> b
-                | None ->  String.Empty
-            let success, errorMsg, actualIL = ILChecker.verifyILAndReturnActual p expectedIL
-
-            if not success then
-                // Failed try update baselines if required
-                // If we are here then the il file has been produced we can write it back to the baseline location
-                // if the environment variable TEST_UPDATE_BSL has been set
-                updateBaseLineIfEnvironmentSaysSo baseline.ILBaseline
-                createBaselineErrors baseline.ILBaseline actualIL
-                let errorMsg = (convenienceBaselineInstructions baseline.ILBaseline expectedIL actualIL) + errorMsg
-                Assert.Fail(errorMsg)
+    let private verifyFSILBaseline (baseline: Baseline option) (result: CompilationOutput) : unit =
```

Comment:
I remember refactoring this so the baseline is not an option (since it is checked upfront at call site), I'm just wondering if this change (beside the adjustment to verifyILAndReturnActual call that this PR brings) is intended?

Reply:
I think, I made the actual changes a long while ago in my accessibility.  I imagine that frequent 
````git pull upstream main --rebase```` caused me to lose your modifications.  I will address this.